runtime/src/system_instruction_processor.rs

```diff
@@ -131,35 +154,39 @@ fn create_account(
     space: u64,
     owner: &Pubkey,
     signers: &HashSet<Pubkey>,
+    invoke_context: &mut dyn InvokeContext,
 ) -> Result<(), InstructionError> {
     // if it looks like the `to` account is already in use, bail
     {
         let to = &mut to.try_account_ref_mut()?;
         if to.lamports > 0 {
-            debug!(
-                "Create Account: invalid argument; account {:?} already in use",
+            ic_msg!(
+                invoke_context,
+                "Create Account: account {:?} already created",
```

Comment:
```suggestion
                "Create Account: account {:?} already in use",
```

To match previous messages